modules/end-user-guide/partials/proc_creating-a-workspace-from-a-feature-branch-of-a-git-repository.adoc

```diff
@@ -15,9 +15,9 @@ A {prod-short} workspace can be created by pointing to devfile that is stored in
 Execute the workspace by opening the following URL: `pass:c,a,q[{prod-url}/#__<GitHubBranch>__]`
 
 .Example
-Use following URL format to open an experimental link:https://github.com/quarkusio/quarkus-quickstarts[quarkus-quickstarts] branch hosted on link:https://che.openshift.io[che.openshift.io].
+Use following URL format to open an experimental link:https://github.com/quarkusio/quarkus-quickstarts[quarkus-quickstarts] branch hosted on link:https://workspaces.openshift.com[workspaces.openshift.com].
 
 [subs="+quotes"]
 ----
-https://che.openshift.io/#https://github.com/maxandersen/quarkus-quickstarts/tree/che
+https://workspaces.openshift.com/#https://github.com/maxandersen/quarkus-quickstarts/tree/che
```

Comment:
@themr0c this is an incorrect reference to factory - should be based on `f?url=` see #1884